common/src/main/java/org/apache/atlas/service/redis/NoRedisServiceImpl.java

```diff
@@ -21,7 +21,7 @@ public void init() {
     @Override
     public boolean acquireDistributedLock(String key) {
         //do nothing
-        return true;
+        return false;
```

Comment:
By changing this what default behaviour would change?

Reply:
This is to avoid getting lock during non redis case. In case we want to debug anything in prod, we dont want all pods to get the lock.